movie/models.py

```diff
@@ -0,0 +1,49 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    name         = models.CharField(max_length=200)
+    contry       = models.CharField(max_length=200)
+    main_image   = models.URLField(max_length=1000)
+    description  = models.TextField(null=True)
+    opening_at   = models.DateField()
+    show_time    = models.IntegerField()
+
+    class Meta:
+        db_table = 'movies'
+
+class Picture(models.Model):
+    movie  = models.ForeignKey('Movie', on_delete=models.CASCADE)
+    url    = models.URLField(max_length=1000)
+
+    class Meta:
+        db_table = 'pictures'
+
+class Actor(models.Model):
+    name           = models.CharField(max_length=200)
+    proflie_image  = models.URLField(max_length=1000)
+    movie          = models.ManyToManyField('Movie', through='MovieActor')
```

Comment:
```suggestion
    movies          = models.ManyToManyField('Movie', through='MovieActor')
```
M2M 필드 선언해주실때는 헷갈리지않게 복수형으로 선언해주세요.